src/frontend/efiling-frontend/src/components/page/cso-account/CSOAccount.js

```diff
@@ -29,59 +29,68 @@ export default function CSOAccount({
   const [termsAccepted, acceptTerms] = useState(false);
   const [continueBtnEnabled, setContinueBtnEnabled] = useState(false);
   const [showLoader, setShowLoader] = useState(false);
-  const [emailInput, setEmailInput] = useState({ email: "", confEmail: "" });
+  const [emailInput, setEmailInput] = useState({
+    email: applicantInfo.email,
+    confEmail: applicantInfo.email,
+  });
```

Comment:
Yeah, good.  I think this was the main bug ... the initial state of the form fields wasn't set.

Reply:
looool